style.css

```diff
@@ -505,7 +507,129 @@ textarea:focus {
 
 /* Contact me ends here */
 
+.detail-card {
+  display: none;
+}
+
+.detail-card--show {
+  display: block;
+  z-index: 15;
+  height: 100%;
+  width: 100%;
+  position: fixed;
+  background-color: rgba(0, 0, 0, 0.75);
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  overflow-y: auto;
+  margin: auto 0;
+}
+
+.detail-card-inner {
```

Comment:
- [ ] The `Live Version` and `Source` buttons should be on a single line to match the template. I think the margin of this container is a bit too much when compared to the design. Please decrease it so the contents inside are aligned properly. 🥸️
- [ ] Please add a thin horizontal line above your buttons as shown in the design template. 🙂️
- [ ] You are using a dark background color while the design is using a lighter one. Please update it. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://github.com/fatima-najafi/Portfolio/assets/17243325/52ba5184-dd87-420f-9d17-269f048c83a0" width="300">|<img src="https://github.com/fatima-najafi/Portfolio/assets/17243325/65850437-6963-49b1-837d-5b8f6176feac" width="300">|